tests/integration/framework/process/daprd/daprd.go

```diff
@@ -186,7 +199,7 @@ func (d *Daprd) WaitUntilAppHealth(t *testing.T, ctx context.Context) {
 			}
 			defer resp.Body.Close()
 			return http.StatusNoContent == resp.StatusCode
-		}, time.Second*10, 100*time.Millisecond)
+		}, 10*time.Second, 100*time.Millisecond)
```

Comment:
Big nit: if we want to be consistent on precision and magnitude ordering, can we have magnitude on the right? This can read better on multiple lines as the column matches up on precious regardless of magnitude, i.e. 

```
12934*time.Second
12*time.Second
```

vs

```
time.Second*12934
time.Second*12
```

Reply:
TBH I prefer the opposite. In natural language, we say "10 seconds". it's also common in the SI to put units after the number, for everything except for currencies in North America ($10 but it's 10€)